tests/runtime/http_server/test_http_server.py

```diff
@@ -198,47 +200,59 @@ def test_mutual_tls_server_with_wrong_cert(open_port):
 
 
 @pytest.mark.parametrize(
-    ["enable_inference", "enable_training"],
-    [(True, False), (False, True), (False, False)],
+    ["enable_inference", "enable_training", "enable_inference_jobs", "error"],
+    [
+        (True, False, False, False),
+        (False, True, False, False),
+        (False, False, False, False),
+        (False, False, True, True),
+    ],
 )
-def test_services_disabled(open_port, enable_inference, enable_training):
+def test_services_disabled(
+    open_port, enable_inference, enable_training, enable_inference_jobs, error
+):
     with temp_config(
         {
             "runtime": {
                 "service_generation": {
                     "enable_inference": enable_inference,
                     "enable_training": enable_training,
+                    "enable_inference_jobs": enable_inference_jobs,
                 }
             },
         },
         "merge",
     ):
-        with runtime_http_test_server(open_port) as server:
-            # start a non-blocking http server with basic tls
-            resp = requests.get(
-                f"http://localhost:{open_port}{http_server.HEALTH_ENDPOINT}",
-            )
-            resp.raise_for_status()
-            assert server.enable_inference == enable_inference
-            assert (
-                server.global_predict_servicer
-                and server.model_management_servicer
-                and enable_inference
-            ) or (
-                server.global_predict_servicer is None
-                and server.model_management_servicer is None
-                and not enable_inference
-            )
-            assert server.enable_training == enable_training
-            assert (
-                server.global_train_servicer
-                and server.training_management_servicer
-                and enable_training
-            ) or (
-                server.global_train_servicer is None
-                and server.training_management_servicer is None
-                and not enable_training
-            )
+        error_context = (
+            pytest.raises(CaikitRuntimeException) if error else nullcontext()
+        )
+        with error_context:
+            with runtime_http_test_server(open_port) as server:
+                # start a non-blocking http server with basic tls
```

Comment:
I know you didn't write it, but I don't _think_ this comment is accurate (or useful). Can you remove it?